### PR TITLE
Adding Technical Charter

### PR DESCRIPTION
**Pull request by redenmartinez**

### CODE REVIEW

**Comment by hythloda** [comment]
@puerco Can you review?

File: docs/CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+viii. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+ix. coordinating any marketing, events, or communications regarding the Project.
+
+## 3. TSC Voting
+
+a. While the Project aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Project forward, the voting members of the TSC will vote on a one vote per voting member basis.
+
+b. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
```
Comment (KennyPaul) [inline docs/CHARTER.md:57]: Concerns raised over the TSC being able to meet quorum on a consistent basis. A couple of options were discussed. @fkautz recommended a common compromise of _members are temporarily dropped from the quorum requirement based upon attendance._   

Typical language I have seen used in several other communities for this model is:

- If a TSC member misses any two consecutive TSC meetings, such member shall not be counted against the quorum for voting purposes until that member next attends a TSC meeting. 

-kenny

File: docs/CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+## 7. Intellectual Property Policy
+
+a. Collaborators acknowledge that the copyright in all new contributions will be retained by the copyright holder as independent works of authorship and that no contributor or copyright holder will be required to assign copyrights to the Project. 
+
+b. Except as described in Section 7.c., all contributions to the Project are subject to the following: 
+
+i. All new inbound code contributions to the Project must be made using Apache License, Version 2.0 available at http://www.apache.org/licenses/LICENSE-2.0  (the “Project License”). 
+
+ii. All new inbound code contributions must also be accompanied by a Developer Certificate of Origin ([http://developercertificate.org](http://developercertificate.org)) sign-off in the source code system that is submitted through a TSC-approved contribution process which will bind the authorized contributor and, if not self-employed, their employer to the applicable license;
```
Comment (KennyPaul) [inline docs/CHARTER.md:99]: Concerns about possibly excluding non-affiliated contributors - Wish to investigate DCO for ICs and CCLA for company contributors.

Comment (KennyPaul) [reply docs/CHARTER.md:99]: discussed Apr. 10 - decision to stick with the DCO as the requirement.

File: docs/CHARTER.md (hunk 1 above)
Comment (KennyPaul) [reply docs/CHARTER.md:57]: discussed Apr. 10 - decision to use the suggested language and rely on electronic voting model when necessary.

**Comment by puerco** [comment]
The community reached consensus to accept the charter on the April 10th community meeting ([see meeting notes](https://docs.google.com/document/d/14mEp4L6DJCVhlxip6yfTsu_34hO2o8eEp8wP5Tn1plI/edit#heading=h.4wdpuu2dxsel)) with the relaxed quorum language clause as the proposed governing model as it matches our current decision process.

File: docs/CHARTER.md (hunk 1 above)
Comment (puerco) [reply docs/CHARTER.md]: I pushed a commit adding the language discussed during the meetting.

**Comment by puerco** [comment]
Thanks @redenmartinez @hythloda and @KennyPaul for your support, and to the protobom community for your valuable input :two_hearts:

**Review by puerco — APPROVED** [review]

**Pull request merged**